model-optimizer/extensions/middle/MXFFTToDFT.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.dft import DFT, IDFT
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.pad import Pad
+from mo.ops.reshape import Reshape
+from mo.ops.squeeze import Squeeze
+from mo.ops.strided_slice import StridedSlice
+from mo.ops.unsqueeze import Unsqueeze
+
+
+class MXFFTToDFT(MiddleReplacementPattern):
```

Comment:
This transformation can be implemented during the front phase. You can use Broadcast operation to broadcast scalar 0 to a tensor with shape Unsqueeze(0, Rank(input) + 1). And then you can use ScatterUpdate to set a particular element to value -1. 

But before you start doing this, could you explain the logic of this transformation? I would like to see what this transformation actually does. How the transformation from MxNet DFT to OpenVINO DFT is performed?

Reply:
Fixed.